cg/cli/workflow/balsamic/base.py

```diff
@@ -91,7 +78,7 @@ def link(context, case_id, sample_id):
             )
         else:
```

Comment:
Flatten this by adding `continue` after if and remove `else`

Reply:
flattened